osu.Game/Screens/Footer/ScreenFooter.cs

```diff
@@ -40,7 +40,11 @@ public partial class ScreenFooter : OverlayContainer
 
         private const int padding = 60;
         private const float delay_per_button = 30;
-        private const double transition_duration = 400;
+        private const double transition_duration = 500;
+
+        // Disable masking because it breaks due to the height of this container being less than the displayed content.
+        // The height being set as it is is required for transition purposes.
+        public override bool UpdateSubTreeMasking() => false;
```

Comment:
I'm not entirely sure what this is supposed to be fixing (as in I don't see anything break when this is commented out), but I suppose I'll just smile and nod to finally move this forward?